CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.6.2
+ - Fix bug where update index actions would not work with events with 'data' field
```

Comment:
this "bug" is a very specific format prerequisite that we should add to the docs